library/src/main/java/com/mux/player/MuxPlayer.kt

```diff
@@ -71,7 +71,20 @@ class MuxPlayer private constructor(
     exoPlayer.addListener(object : Listener {
       // more listener methods here if required
       override fun onMediaItemTransition(mediaItem: MediaItem?, reason: Int) {
-        //muxStats?.videoChange(CustomerVideoData())
+        // Check if a DRM token is set, set View Drm Type if it is
+        // TODO: escape hatch TBA
+        if (mediaItem.requestMetadata.extras.getString(Constants.BUNDLE_DRM_TOKEN) != nil) {
+          val viewData = CustomerViewData()
+          // Assumes only widevine DRM playback is supported
+          // If playready support is added in future, update to select between widevine and playready
+          viewData.viewDrmType = Constants.VIEW_DRM_TYPE_WIDEVINE
+
+          // TODO: Confirm this doesn't overwrite other keys like view session ID to null
```

Comment:
Confirmed, the existing customer data is preserved. There's several hops through the core, but eventually `BaseQueryData.update()` is called, [implementation here](https://github.com/muxinc/mux-stats-sdk-java/blob/452e404b5d52b5ea7a58ca5d140f3860a06b3c01/MuxCore/src/main/java/com/mux/stats/sdk/core/model/BaseQueryData.java#L44).

As you can see, only keys present in the input `.*WhateverData` are set on the existing object.